kaggle_environments/envs/werewolf/test_werewolf.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+
+from kaggle_environments import make
+
+URLS = {
+    "gemini": "https://logos-world.net/wp-content/uploads/2025/01/Google-Gemini-Symbol.png",
+    "openai": "https://images.seeklogo.com/logo-png/46/1/chatgpt-logo-png_seeklogo-465219.png",
+    "claude": "https://images.seeklogo.com/logo-png/55/1/claude-logo-png_seeklogo-554534.png",
+    "grok": "https://images.seeklogo.com/logo-png/61/1/grok-logo-png_seeklogo-613403.png",
+}
+
+
+@pytest.fixture
+def agents_config():
+    roles = ["Werewolf", "Werewolf", "Doctor", "Seer", "Villager", "Villager", "Villager"]
+    names = [f"player_{i}" for i in range(len(roles))]
+    thumbnails = [
+        URLS["gemini"],
+        URLS["gemini"],
+        URLS["openai"],
+        URLS["openai"],
+        URLS["openai"],
+        URLS["claude"],
+        URLS["grok"],
+    ]
+    agents_config = [
+        {"role": role, "id": name, "agent_id": "random", "thumbnail": url}
+        for role, name, url in zip(roles, names, thumbnails)
+    ]
+    return agents_config
+
+
+@pytest.fixture
+def env(agents_config):
+    env = make("werewolf", debug=True, configuration={"agents": agents_config})
+    return env
+
+
+def test_load_env(env):
+    agents = ["random"] * 7
+    env.run(agents)
+
+    for i, state in enumerate(env.steps):
+        env.render_step_ind = i
+        env.renderer(state, env)
+
+
+def test_discussion_protocol(agents_config):
+    env = make(
+        "werewolf",
+        debug=True,
+        configuration={
+            "agents": agents_config,
+            "discussion_protocol": {"name": "RoundRobinDiscussion", "params": {"max_rounds": 2}},
+        },
+    )
+    agents = ["random"] * 7
+    env.run(agents)
+    env.toJSON()
+
+
+def test_no_reveal_options(agents_config):
+    env = make(
+        "werewolf",
+        debug=True,
+        configuration={
+            "agents": agents_config,
+            "night_elimination_reveal_level": "no_reveal",
+            "day_exile_reveal_level": "no_reveal",
+        },
+    )
+    agents = ["random"] * 7
```

Comment:
@hannw I'd love to see more tests and more asserts in the tests. The existing suite seems reasonable solely for confirming that the game runs to completion. Refactoring kaggle-environments seems to pose a high risk of silently altering game behavior at the moment.